rdwatch/core/tasks/animation_export.py

```diff
@@ -304,6 +304,7 @@ class GenerateAnimationSchema(BaseModel):
     include: list[Literal['obs', 'nonobs']] = ['obs', 'nonobs']
     rescale: bool = False
     rescale_border: int = Field(1, ge=0)  # Border rescale should be an integer >= 0
+    line_thickness_factor: float = 1.0
```

Comment:
This same stuff needs to be added to the ./rdwatch/scoring/tasks/animation_export.py to maintain parity.

Reply:
Added to `scoring/animation_export.py`.